reducers/gotPlayerMatches.js

```diff
@@ -0,0 +1,30 @@
+import { playerMatchesActions } from '../actions';
```

Comment:
yea they can be separate reducers which are combined under gotPlayer so we don't have some mammoth reducer file. So like gotPlayer.matches or gotPlayer.otherStuff